Makefile

```diff
@@ -84,7 +84,7 @@ validate:
 
 		echo terraform-validate: $$1 succeeded
 		exit $$EXIT_CODE
-	}
+	)}
```

Comment:
Do you really need a subshell? Functions will return the exit code from the last line and a script will terminate if a function returns non-zero if `set -e`. Generally speaking, bash script should always start with `set -eu` (and probably `set -o pipefail`) and should not have to turn any of them off.